frontend/components/MobileAccordion.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="p-4">
+    <!-- Mobile View -->
+    <div class="md:hidden">
+      <div
+        v-for="(item, menuKey) in menuItems"
+        :key="menuKey"
+        class="border-b-2 border-gray-200 py-2"
+      >
+        <button
+          class="w-full flex justify-between items-center"
+          @click="toggleItem(item.id)"
+        >
+          <span class="font-bold italic text-offblack">{{ item.title }}</span>
+          <span class="text-xl px-2">
+            {{ isActive(item.id) ? '-' : '+' }}
+          </span>
+        </button>
+        <transition name="fade">
```

Comment:
For the sake of simplicity, I would probably opt in for custom transition here (`<transition name="accordion">`) instead of the Vue's default transition. The reason for this is that we can get granular control and have it act like accordion on mobile, but it wouldn't have any effect whatsoever on desktop.

Reply:
updated